projects/com.oracle.truffle.llvm.test/src/com/oracle/truffle/llvm/test/LLVMPaths.java

```diff
@@ -75,6 +75,9 @@ private LLVMPaths() {
     // Interop test suites
     public static final String INTEROP_TESTS = LOCAL_TESTS + "/../interoptests";
 
+    // InlineAssembly test suite
+    public static final File INLINEASSEMBLY_TESTS = new File(LOCAL_TESTS + "/c/inlineassembly");
```

Comment:
If you have a separate test suite for inline assembly tests, then it should not be in the `LOCAL_TESTS`, since otherwise the test cases are executed by the original and the new test suite.